src/server_manager/install_scripts/install_server.sh

```diff
@@ -378,14 +378,18 @@ install_shadowbox() {
   readonly SB_IMAGE=${SB_IMAGE:-quay.io/outline/shadowbox:stable}
 
   log_for_sentry "Setting PUBLIC_HOSTNAME"
-  # TODO(fortuna): Make sure this is IPv4
   PUBLIC_HOSTNAME=${FLAGS_HOSTNAME:-${SB_PUBLIC_IP:-$(curl -4s https://ipinfo.io/ip)}}
 
   if [[ -z $PUBLIC_HOSTNAME ]]; then
```

Comment:
Let's check the result code from curl. Relying on the variable being set is not good.

Reply:
Thanks to `set -e` and `--fail`, we actually cannot check the result code, because a non-success result code terminates the script.  (Actually, we *are* checking the result code, as `$EXIT_CODE` in `finish()`.)

Reply:
That's right. That's good enough.